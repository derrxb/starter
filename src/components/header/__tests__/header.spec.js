import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';

describe('Header Spec', () => {
  it('renders correctly', () => {
    // Arrange
    const links = [
      {
        name: 'About',
        to: '/about',
      },
      {
        name: 'Contact',
        to: '/book',
      },
    ];

    // Act
    const tree = renderer.create(<Header links={links} />).toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
  });
});
