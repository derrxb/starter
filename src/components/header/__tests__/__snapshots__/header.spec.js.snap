// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Spec renders correctly 1`] = `
<nav
  className="sc-ifAKCX izVqPn"
>
  <a
    className="sc-EHOje ffZhMT"
    href="/"
    nature="default"
  >
    
  </a>
  <div
    className="sc-bZQynM iFlhOA"
  >
    <a
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      className="sc-bdVaJa cHnYwO"
      data-testid="nav-link"
      href="/about"
      nature="default"
    >
      About
    </a>
    <a
      activeStyle={
        Object {
          "textDecoration": "underline",
        }
      }
      className="sc-bdVaJa cHnYwO"
      data-testid="nav-link"
      href="/book"
      nature="default"
    >
      Contact
    </a>
  </div>
  <button
    className="sc-bwzfXH fewVcK"
    onClick={[Function]}
  >
    Menu
  </button>
</nav>
`;
